vhost/src/vhost_user/frontend.rs

```diff
@@ -935,7 +932,8 @@ mod tests {
 
         let pfeatures = VhostUserProtocolFeatures::all();
         let hdr = VhostUserMsgHeader::new(FrontendReq::GET_PROTOCOL_FEATURES, 0x4, 8);
-        let msg = VhostUserU64::new(pfeatures.bits());
+        // Unknown feature bits should be ignored.
+        let msg = VhostUserU64::new(pfeatures.bits() | 1 << 63);
```

Comment:
I'm not sure the comment is enough, can a const be created?, with some descriptive name like `const FAKE_FEATURE: u64 = 1 << 63`  instead, because it could be interpreted that setting that bit, enables a feature to ignore features.

besides that LGTM

Reply:
Done.